src/pudl/analysis/record_linkage/eia_ferc1_model_config.py

```diff
@@ -19,7 +18,7 @@
 blocking_rule_7 = "l.report_year = r.report_year and l.capacity_mw = r.capacity_mw and substr(l.plant_name_mphone,1,2) = substr(r.plant_name_mphone,1,2)"
 blocking_rule_8 = "l.report_year = r.report_year and l.installation_year = r.installation_year and substr(l.plant_name_mphone,1,2) = substr(r.plant_name_mphone,1,2)"
 blocking_rule_9 = "l.report_year = r.report_year and l.construction_year = r.construction_year and substr(l.plant_name_mphone,1,2) = substr(r.plant_name_mphone,1,2)"
-blocking_rule_10 = block_on(["report_year", "net_generation_mwh"])
+blocking_rule_10 = block_on("report_year", "net_generation_mwh")
```

Comment:
Change in the function signature. Takes an arbitrary number of positional args now.